exporter/awsxrayexporter/internal/translator/aws_test.go

```diff
@@ -99,10 +95,10 @@ func TestAwsFromEcsResource(t *testing.T) {
 
 	assert.NotNil(t, filtered)
 	assert.NotNil(t, awsData)
-	assert.NotNil(t, awsData.EC2)
 	assert.NotNil(t, awsData.ECS)
+	assert.Nil(t, awsData.EC2)
```

Comment:
Hmm... I wonder if this will break any customers existing trace data, because we are now more strict about what metadata we record. IMO it is more correct this way, but just to make sure we're thinking it through: should we allow customers to record e.g. EC2 metadata even though they're on another platform (e.g. EKS)?

Reply:
By doing this, we're giving the correct Origin to customer. 
In another way to say, will customer be happy if we tell them their service is running on EC2 but it is actually running on ECS? :)

But do you know if it will introduce the broken experience to the existing x-ray service map if we changed the Origin on the existing setting?